account_financial_report/report/templates/trial_balance.xml

```diff
@@ -27,40 +27,123 @@
                     <div class="act_as_table data_table" style="width: 100%;">
                         <!-- Display account header -->
                         <t t-call="account_financial_report.report_trial_balance_lines_header"/>
-
+                        
                         <!-- Display each lines -->
                         <t t-foreach="o.account_ids" t-as="line">
+                            <t t-set="type" t-value='"account_type"'/>                                                                    
+                            <!-- Adapt -->
+                            <t t-set="style" t-value="'font-size:8px;'"/>
+                            <t t-if="line.account_group_id and len(line.code) == 1">
+                                <t t-set="style" t-value="'font-size: 11px; font-weight: bold; color: blue;'"/>
+                                <t t-set="padding" t-value="0"/>
+                            </t>
+                            <t t-if="line.account_group_id and len(line.code) == 2">
+                                <t t-set="style" t-value="'font-size: 10px; font-weight: bold; color: blue;'"/>
+                                <t t-set="padding" t-value="4"/>
+                            </t>
+                            <t t-if="line.account_group_id and len(line.code) == 3">
+                                <t t-set="style" t-value="'font-size: 9px; font-weight: bold; color: blue;'"/>
+                                <t t-set="padding" t-value="8"/>
+                            </t>
+                            <t t-if="line.account_group_id and len(line.code) == 4">
+                                <t t-set="style" t-value="'font-size: 9px; font-weight: bold; color: blue;'"/>
+                                <t t-set="padding" t-value="12"/>
+                            </t>
+                            <t t-if="line.account_group_id and len(line.code) == 5">
```

Comment:
I suggest that instead of hardcoding we set a account.financial.report.style model that allows to define the style to apply to each level, similar to how it is done in MIS Builder.

Reply:
@jbeficent I agree with the style, but probably will be easier to have a destinated module for styling on reporting-engine, so it can be inheritable...

Reply:
@jbeficent @OCA/mis-builder-maintainers @OCA/reporting-engine-maintainers I can try to separate the style from mis_builder module into a separate module in reporting-engine and in mis_builder and  account_financial_report to use the style from newly created module. What do you think?

Reply:
@jbeficent I suggest to leave it like this, until we will have a decision.